projects/api-design-guidelines/docs/sorting.md

```diff
@@ -3,11 +3,18 @@
 Endpoints that support sorting based on one or more field names and order(s) must use a `sort` query parameter with the following syntax:
 
 ```
-?sort[field_name_1]=desc&sort[field_name_2]=asc
```

Comment:
I would keep this under a separate header at the end, for example "Deprecated syntax" and mention that this was the old way but that it was changed for the new method due to issues with the order not always being easy to determine in various programming languages. It can also mention that APIs that implemented the old sort method should keep doing so for BC, but should also implement the new method. (In reality probably Search API, maybe some other small "search" endpoints in Entry API or others)